components/projects/NewProject/NewProjectForm.tsx

```diff
@@ -29,9 +30,32 @@ function NewProjectForm({ employees, customers }: NewProjectFormProps): JSX.Elem
         }
     };
 
-    const handleSumbit = (e: React.FormEvent<HTMLFormElement>) => {
+    const handleSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
         e.preventDefault();
-        console.log(state);
+        // console.log(state);
+        const data = {
```

Comment:
Can you infer the type of `data` ? Also the variable could have a better name. `createProjectRequest` or something that gives a hint what it's used for
```typescript
const createProjectRequest: ProjectDTO = {
....
}
```

Reply:
Yes. And in case there's a problem with using all the fields (probably won't be), you can always use a partial type:

`const createProjectRequest: Partial<components['schemas']['ProjectDTO']> = { ... }`

Reply:
Done in 5bf47e0